samples/highcharts/website/icon-chart/demo.js

```diff
@@ -117,52 +160,90 @@ const charts = {
                         visible: true
                     });
 
+                    for (let gg = 0; gg < gridLine.length; ++gg) {
+                        gridLine[gg].style.stroke = 'transparent';
+                    }
+                    cover.style.fill =  '#30426B';
+                    background.style.fill = '#f0f0f0';
+                    plotBackground.style.transition = 'fill 2s';
+                    plotBackground.style.fill = '#f0f0f0';
 
-                    done = true;
-                    $('#charts .highcharts-grid-line').css({ stroke: 'transparent' });
-                    $('.cover').css({ fill: '#30426B' });
-                    $('.highcharts-background').css({ fill: '#f0f0f0' });
-                    $('.highcharts-plot-background').css({ fill: '#f0f0f0', transition: 'fill 2s' });
-                    $('.particle').each(function () {
-                        $(this).addClass('iceberg');
-                    });
+                    for (let pp = 0; pp < particle.length; ++pp) {
+                        particle[pp].classList.add('iceberg');
+                    }
 
                 }, 5000);
+                setTimeout(function () {
+                    for (let pp = 0; pp < particle.length; ++pp) {
+                        particle[pp].classList.add('clip');
+                    }
+                    for (let hh = 14; hh <= 22; ++hh) {
+                        if (hh % 2 === 0) {
+                            chart.series[hh].update({
+                                visible: true
+                            });
+                        }
+                    }
+                    if (reduced) {
+                        particles.style.transition = 'opacity 1s';
+                        particles.style.opacity = 1;
+                    }
+                    bergDepth1.style.opacity = 1;
+                    bergDepth2.style.opacity = 1;
+                    bergDepth3.style.opacity = 1;
+                    bergDepth4.style.opacity = 1;
+                    bergDepth5.style.opacity = 1;
```

Comment:
No need to adress the `berg-depth` elements one by one. This jQuery statement

`$('#charts .berg-depth').css({ opacity: 0.6 });`

... can be transformed directly (and more readable) by using `querySelectorAll`. 

```js
[].forEach.call(
    document.querySelectorAll('#charts .berg-depth'),
    elem => elem.style.opacity = 0.6
);
```